regen_perly.pl

```diff
@@ -76,10 +76,10 @@
 
 # Don't change this to add new bison versions without testing that the generated
 # files actually work :-) Win32 in particular may not like them. :-(
-unless ($version =~ /\b(1\.875[a-z]?|2\.[0134567]|3\.[0-4])\b/) { die <<EOF; }
+unless ($version =~ /\b(1\.875[a-z]?|2\.[0134567]|3\.[0-7]+)\b/) { die <<EOF; }
```

Comment:
Did you mean to include this `+` at the end?

Reply:
I originally made it `[0-9]+` to allow any 3.x version, but given the amount of work to get from 3.4 to 3.6, I discarded that idea as unwarrantedly optimistic, but failed to completely fix it.